docs/newuser.md

```diff
@@ -1,29 +1,41 @@
 
-## Office 365 Account
-[Office 365](https://portal.office.com), these are provided by emailing DigitalCapabilitiesPMO@homeoffice.gsi.gov.uk, this is not managed by us.
+## Internally Accessible
 
 ### Artifactory
-[Artifactory](https://artifactory.digital.homeoffice.gov.uk.). Sign in with office 365 button. The docker repository in artifactory is accessed by the docker.digital.homeoffice.gov.uk address.
+[Artifactory](https://artifactory.digital.homeoffice.gov.uk.). Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview) Sign in with Office 365 button. The docker repository in artifactory is accessed by the docker.digital.homeoffice.gov.uk address.
+
 ### Gitlab
-[GitLab](https://gitlab.digital.homeoffice.gov.uk) To get access to this you will need to first login via office 365 by clicking the office 365 login button and and then request for your account to be unblocked.
-## VPN
-The VPN is used to be able to get through to our multiple AWS environments as well as other data centers. The profiles can be downloaded from [authd](https://authd.digital.homeoffice.gov.uk) which you can access using your office 365 SSO. The following services can be only accessed when connected to the Dev VPN profile.
+[GitLab](https://gitlab.digital.homeoffice.gov.uk) To get access to this you will need to first login via office 365 by clicking the office 365 login button requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview)
+
 ### Kubernetes
-You will need a token before you can access any of the clusters. These can be requested on the [bau board](https://github.com/UKHomeOffice/application-container-platform-bau/issues).
+You will need a token before you can access any of the clusters. Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview) 
+
 ### Drone
 We have two instances [gitlab](https://drone-gitlab.digital.homeoffice.gov.uk) uses your gitlab login and the one for [github](https://drone.digital.homeoffice.gov.uk) uses your github login.
+
 ### Kibana
-[Kibana](https://kibana.ops.digital.homeoffice.gov.uk) uses office365 sigin.
+[Kibana](https://kibana.ops.digital.homeoffice.gov.uk) uses Office 365 sign in.
+
 ### Sysdig
-[Sysdig](https://sysdig.digital.homeoffice.gov.uk). You need to request an account on our [bau board](https://github.com/UKHomeOffice/application-container-platform-bau/issues) if you want access.
+[Sysdig](https://sysdig.digital.homeoffice.gov.uk). Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview) 
+
 ## AWS
-Access to AWS is managed by the [Hoddat-iam](https://gitlab.digital.homeoffice.gov.uk/Devops/hoddat-iam) repo.
+Access to AWS is also managed by the Platform Hub. Requests can be made on the [Platform Hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview)
```

Comment:
Access to AWS is not quite "managed by the Platform Hub" – I think it's a case of using the "general support" request form to ask the ACP team to grant access?